readme.md

```diff
@@ -155,11 +156,15 @@ Marker `VALUE` specifies a property value e.g.: `red` => `[VALUE, 'red']`.
 
 #### Values list
 
+`[VALUE, 'red']+`
+
 A comma separated list of simple values e.g.: `red, green` => `[VALUE, 'red'], [VALUE, 'green']`.
 
 
 #### Compound value
 
+`[COMPOUND_VALUE_START], [VALUE, <value>]+, [COMPOUND_VALUE_END]`
```

Comment:
Hmm, I'm not sure why we'd need to handle space-separated compound variables any differently to anything else? These occur all the time, margin/padding, box-shadow, animation etc. Might cause some unnecessary overhead.

However, comma-separated compound variables definitely seems like a good point to break, but I don't see that catered for anywhere yet.